README.md

```diff
@@ -1,2 +1,117 @@
+* **rovr_gazebo**		gazebo simulation environment
+* **rovr_input**		testing input/movement
+* **rovr_navigation**	navigation, flexbe robot behaviors
+* **decawave**		localization using decawaves
+* **formatter_string**	string formatter
+* **navx**			navx IMU data
+* **socketcan_bridge**	socketcan bridge
+* **socketcan_interface**	socketcan interface
+* **telecom**			telecom
+
+## Other Repos
+### Companion Repos
+ - [serial](https://github.com/GOFIRST-Robotics/serial)
+ - [realsense-ros](https://github.com/GOFIRST-Robotics/realsense-ros)
+- [aruco_localization](https://github.com/GOFIRST-Robotics/aruco_localization)
+- [navXTimeSync](https://github.com/GOFIRST-Robotics/navXTimeSync)
+### Previous Years
+
+ - [NASA-RMC-2019-NorthstarRobotics](https://github.com/GOFIRST-Robotics/NASA-RMC-2019-NorthstarRobotics)
+- [NASA-RMC-2018](https://github.com/GOFIRST-Robotics/NASA-RMC-2018)
+
+## Resources
+### Communication
+All team communication is through Slack - please see an officer or team member for instructions on how to joing our Slack workspace.  Relevant channels are "rmc" and "rmc_programming".
+
+
+## FAQs
+### NASA FAQ
+[FAQ](https://www.nasa.gov/sites/default/files/atoms/files/rmc_lunabotics_2020_faq_03.pdf)
+### How do I get the Intel RealSense to work?
+Follow the installation instructions at this [repo](https://github.com/IntelRealSense/realsense-ros).
+### How do I get started with Linux?
+Here are some recommendations from Jude:
+[Single Page Command Line Essentials](https://drive.google.com/file/d/1d_TEG5M8cbDhSmptYmyXXeNGSyzCbaeC/view)  
+[link](https://drive.google.com/file/d/1chCfI9dKEk5xn1EhsuBepCiudX6nKlng/view)  
+[link](https://drive.google.com/file/d/1erAJl0C8ypFN3QjhTqR0VGCr4sjenJmp/view)
+Good chapters: 2-6, 10, 12, 15, 17  
+[link](https://drive.google.com/file/d/1VJa_LGtTaZmOy9H4unzVqFZFCB_CC14B/view)
+### What is ROS?
+[Wikipedia](https://en.wikipedia.org/wiki/Robot_Operating_System): 
+
+> Robot Operating System (ROS or ros) is robotics middleware (i.e. collection of software frameworks for robot software development). Although ROS is not an operating system, it provides services designed for a heterogeneous computer cluster such as hardware abstraction, low-level device control, implementation of commonly used functionality, message-passing between processes, and package management. Running sets of ROS-based processes are represented in a graph architecture where processing takes place in nodes that may receive, post and multiplex sensor data, control, state, planning, actuator, and other messages.
+
+With ROS, we can use existing packages, share our packages, and facilitate communication between the parts of our software
+
+### Example ROS filesystem
+### Example node communication
+
+### Getting Started with ROS
+-   ROS requires* Ubuntu OS (each version of ROS requires* a specific version of Ubuntu)
+	-   Recommended:
+	    -   Install Ubuntu 18.04  http://releases.ubuntu.com/18.04/
```

Comment:
Remember to do `sudo apt-get update, sudo apt-get upgrade` (separate commands) to update computer post-installation, before installing ros.